plugins/bigcommerce/src/data-plugin.ts

```diff
@@ -0,0 +1,129 @@
+import { APIOperations, ResourceType } from '@builder.io/data-plugin-tools';
+import appState from '@builder.io/app-context';
+import { CommerceAPIOperations } from '@builder.io/commerce-plugin-tools';
+import pkg from '../package.json';
+type BigCommerceResourceType = 'product' | 'category';
+import { BuilderStore, BuilderStoreContext } from '@builder.io/react';
+import { useContext, useEffect } from 'react';
```

Comment:
some of those import are not used here, please remove, ( strange that the linter didn't detect it )